ui/src/components/shared/dialogs/DirectorySearchDialog.vue

```diff
@@ -38,13 +38,15 @@
             :show-filters="showFilters"
             :show-quick-type-filters="showQuickTypeFilters"
             :quick-type-filters-allowed="quickTypeFiltersAllowed"
+            :default-quick-type-filter="defaultQuickTypeFilter"
```

Comment:
Confused why this exists. Should either set quickTypeFiltersAllowed or use the built in defaults...